code/modules/clothing/shoes/colour.dm

```diff
@@ -76,6 +76,11 @@
 	permeability_coefficient = 0.01
 	item_color = "white"
 
+/obj/item/clothing/shoes/white/enforcer
+	name = "hypernobilium weave shoes"
```

Comment:
The wizard version doesnt get shoes? And are you sure about leaving it here instead of artefact.dm?

Reply:
The wizard version has sandals. These don't do anything special except be magical with a fancy description